int32 serialID

bool manual
int32 manualDrive
int32 manualTurn

float32 driveDist
int32 turn
int32 dig
bool dump
bool packin
bool eStop
bool pause
bool cancel
